d/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM debian:bookworm AS build
+FROM debian:trixie AS build
```

Comment:
mmh, not sure we should use trixie here @cyrusmsk 

using a stable version is closest to real world usage for me

Reply:
Other langs are using their own Docker images (which not always could based in stable). But we can revert to bookworm or I can try to use D-specific image (if I find one that works)

Reply:
Yes, I understand that every image could not be based on debian stable, but it's always a distribution considered as stable.

I would prefer to use the last stable, or better an official D docker image, if it exists